test/unified_format.py

```diff
@@ -205,8 +222,14 @@ def get_events(self, event_type):
         return [e for e in self.events if 'Command' not in type(e).__name__]
 
     def add_event(self, event):
-        if type(event).__name__.lower() in self._event_types:
+        event_name = type(event).__name__.lower()
+        if event_name in self._event_types:
             super(EventListenerUtil, self).add_event(event)
+        for id in self._event_mapping[event_name]:
+            if id in self.entity_map:
```

Comment:
I think this "if" is always try and the else is unreachable right? If an event name maps to an id then we know for a fact that the id already exists in entity_map.

Reply:
What if it's the first event? Though removing these lines does not cause the tests to fail.